### ISSUE TITLE
Support for Capybara 3.x

### ISSUE THREAD

**Issue by manuelmeurer** [blowmage/minitest-rails-capybara#45]
Hey, thanks for writing this fantastic gem! 😄 
Are there any plans to support Capybara v3? I tried simply updating the dependency (und patching `minitest-capybara` to support Capybara v3 as well) and it worked fine.
If you could point me into the right direction to what would need to be done to support Cabybara v3, I'd be happy to work on it!

**Comment by MrHubble** [comment]
>I tried simply updating the dependency (und patching minitest-capybara to support Capybara v3 as well) and it worked fine.

Do you have a fork I could use or refer to for how to do this? Thanks.

**Comment by jrmhaig** [comment]
Slightly related to this; If you have this in your Gemfile:

```ruby
gem 'minitest-rails-capybara'
gem 'capybara'
```

(without specifying versions) and do a `bundle update` then your version of `minitest-rails-capybara` will drop down to version 0.0.2, being the most recent version that claims to support any version of `capybara`. The latest version is pinned to `~> 2.7`.

I don't know what the correct solution is here.

**Comment by manuelmeurer** [comment]
@jrmhaig That sounds more like an issue with Bundler than with this gem to me.

@MrHubble Sure, here's my fork: https://github.com/krautcomputing/minitest-rails-capybara/tree/2.x_with_capybara_3

**Comment by jrmhaig** [comment]
Yes and no. Bundler will try to find the most recent version of all gems that satisfy all dependencies. It is therefore possible, according to the definitions of the gems, to upgrade `capybara` by downgrading `minitest-rails-capybara` to 0.0.2. This, I believe, is the correct behaviour for Bundler.

**Comment by MrHubble** [comment]
Perhaps this gem is no longer actively maintained because Rails 5.1 has system tests (with Capybara) baked in? I believe the author of The Minitest Cookbook uses the default system tests setup:

>So far I’m loving this combination of (no) setup and (more) features that system tests provide. If you’ve read about my previous setup based on minitest-rails-capybara in The Minitest Cookbook, let me be clear: this is my new jam, and I’ll be shipping an update to the book sometime after Rails 5.1 officially ships that covers system tests.

https://chriskottom.com/blog/2017/04/full-stack-testing-with-rails-system-tests/

**Comment by manuelmeurer** [comment]
Ah, good to know!
I'm stuck with my app on Rails 4.2 for a while, so it would be great to be able to use Capybara v3 with this gem.
@blowmage What do you think?

**Comment by CyborgMaster** [comment]
@manuelmeurer, I'm currently using your fork to support capybara 3 with Rails 4.2.  Thanks!

**Comment by manuelmeurer** [comment]
@CyborgMaster Glad it works for you! 😄
